DDDEastAnglia/Helpers/Security/IOAuthClientInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using DotNetOpenAuth.AspNet;
+using Microsoft.Web.WebPages.OAuth;
+
+namespace DDDEastAnglia.Helpers.Security
+{
+    public interface IOAuthClientInfo
```

Comment:
Ok. I still think it would benefit from being split into a number of interfaces though.